### ISSUE TITLE
Show an indicator if the cart is exported

### ISSUE THREAD

**Issue by galakhyati** [TycheSoftwares/Salesforce-CRM-for-Abandoned-Cart#30]
We should show an indicator that the abandoned cart is exported to Salesforce CRM.

A sign or a tick, and hide the Export to Salesforce CRM link on hover effect so that the record cannot be exported to Salesforce CRM again.

**Comment by apeksha10** [comment]
Yes.. we should show an indicator that the abandoned cart is exported to Salesforce CRM.

Like we can add notes or we can add information in detail popup.
